src/utils/getShortLang.js

```diff
@@ -0,0 +1,5 @@
+"use strict";
+
+module.exports = lang => {
+    return lang.split("-")[0];
+};
```

Comment:
Don't know how useful this utility is. It is only used once, and in feed.njk the split is just done directly.